README.md

```diff
@@ -98,7 +98,7 @@ Show only relevant data when loading via server. Fold your content to save bytes
 
 
 ##### Page Caching
-Well now you can cache a page in SSR. Pretty simple. Just add cache option to route.  
+We'll now you can cache a page in SSR. Pretty simple. Just add cache option to route.  
 `{ cache: { enable: true, duration: 10000}}`
```

Comment:
Here it will come "Well, now you can cache..."

Reply:
Ah, right!
Would you prefer a new PR or should I push an additional commit?

Reply:
Whatever you feel fine.